dash/development/component_generator.py

```diff
@@ -11,6 +11,7 @@
 import functools
 
 import pkg_resources
+import yaml
```

Comment:
@rpkyle  I think the performance import pattern in official yaml doc is still valid.

```
try:
    from yaml import CLoader as Loader, CDumper as Dumper
except ImportError:
    from yaml import Loader, Dumper
```